app/components/atoms/contributors/Create-Contributors/components/Role.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
```

Comment:
this should be in the `pages` directory, under the specific entity, i.e. `contributors`